recoverytool/src/main.rs

```diff
@@ -187,9 +182,13 @@ async fn main() -> anyhow::Result<()> {
             ]);
 
             let db = Database::new(RocksDb::open(db).expect("Error opening DB"), decoders);
-            let mut dbtx = db.begin_transaction().await;
+            let _dbtx = db.begin_transaction().await;
 
-            let mut change_tweak_idx: u64 = 0;
+            let _change_tweak_idx: u64 = 0;
+
+            unimplemented!();
```

Comment:
? Is there a TODO for it?

Reply:
We can no longer recover funds from a federation until this is impl. or do I misunderstand?

Reply:
Correct, no TODO yet.